test/LinkerSections/function_sections.swift

```diff
@@ -1,10 +1,13 @@
 // REQUIRES: OS=linux-gnu || OS=freebsd
 // RUN: %empty-directory(%t)
 // RUN: %target-build-swift -Xfrontend -function-sections -emit-module -emit-library -static -parse-stdlib %S/Inputs/FunctionSections.swift
-// RUN: %target-build-swift -Xlinker --gc-sections -Xlinker -Map=%t/../../FunctionSections.map -I%t/../.. -L%t/../.. -lFunctionSections %S/Inputs/FunctionSectionsUse.swift
-// RUN: %FileCheck %s < %t/../../FunctionSections.map
+// RUN: %target-build-swift -Xlinker -v -Xlinker --gc-sections -Xlinker -Map=%t/../../FunctionSections.map -I%t/../.. -L%t/../.. -lFunctionSections %S/Inputs/FunctionSectionsUse.swift 2>&1 | sed 's/.*\(gold\|LLD\).*/\1/g' | tr "[:lower:]" "[:upper:]" > %t/../../Linker.txt
```

Comment:
This change takes the stdout and stderr of `%target-build-stdlib` pass __each line__ through `sed` that replaces lines that say either `gold` or `LLD` with just those word, and convert everything to upper case. Then the result of all that is passed as `--check-prefix`?

What happens when `-v` prints more than one line? All the lines will be printed and passed to as extra arguments to `%FileCheck`?

```
$ echo -e 'foo\nbar\nthis will say LLD\nbaz' | sed 's/.*\(gold\|LLD\).*/\1/g'                                                                                                                      foo
bar
LLD
baz
```

Am I missing something?

Reply:
#73470 73470